app/src/main/java/cm/aptoide/pt/download/DownloadAnalytics.java

```diff
@@ -188,15 +180,19 @@ public void startProgress(Download download) {
         .setHadProgress(true);
   }
 
-  public void installClicked(ScreenTagHistory previousScreen, ScreenTagHistory currentScreen,
-      String id, String packageName, String trustedValue, String editorsBrickPosition,
-      InstallType installType, AnalyticsManager.Action action, String previousContext,
-      String currentContext) {
-    editorsChoiceDownloadCompletedEvent(previousScreen, id, packageName, editorsBrickPosition,
+  public void installClicked(String md5, String packageName, String trustedValue,
+      String editorsBrickPosition, InstallType installType, AnalyticsManager.Action action) {
+    ScreenTagHistory previousScreen = navigationTracker.getPreviousScreen();
+    ScreenTagHistory currentScreen = navigationTracker.getCurrentScreen();
+    String previousContext = navigationTracker.getCurrentScreen()
```

Comment:
Use navigationTracker.getViewName(true) and navigationTracker.getVIewName(false) for the current and previous context as it’s how it’s being done in other analytics, just for consistency purposes.

Reply:
This is now fixed. The ScreenTagHistory is still being used in one case, where the tag is needed